great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -833,6 +846,82 @@ def _get_rules_as_dict(self) -> Dict[str, Rule]:
         rule: Rule
         return {rule.name: rule for rule in self.rules}
 
+    # noinspection PyUnusedLocal
+    @staticmethod
+    def _apply_runtime_environment(
```

Comment:
I do appreciate this improvement, thank-you Alex!

Reply:
@talagluck I understand about making the arguments more apparent.  I am going to try -- not sure yet; I am struggling with the issue of making this API call maintainable when we add new `DomainBuilder` classes with their own directives.  I will keep you posted.